src/java/org/apache/cassandra/tools/StandaloneVerifier.java

```diff
@@ -188,10 +196,16 @@ public static Options parseArgs(String cmdArgs[])
                 }
 
                 String[] args = cmd.getArgs();
-                if (args.length != 2)
+                if (args.length < 2)
```

Comment:
Does this change buy you anything?

Reply:
Arguably neutral. It's vestigial from when I had force as a positional argument rather than a - flagged optional and was tinkering a bit, and I ended up leaving it because I liked it a touch more. We could go back to != 2 and ternary if you have strong feelings about it.